api/src/main/java/io/minio/DisableVersionBucketArgs.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio;
+
+/** Argument class of @see #disableVersioning(DisableVersionBucketArgs args). */
+public class DisableVersionBucketArgs extends BucketArgs {
```

Comment:
1. We would need to have name consistency with API name. Here, `DisableVersioningArgs` and `EnableVersioningArgs` are appropriate.
2. Remove any obvious comments for getters (if any).
3. Follow consistent comments for `Args` and `Builder` class. https://github.com/minio/minio-java/pull/934 can be taken as reference.